src/v0.3/parseExpr_0.3.jl

```diff
@@ -132,26 +132,67 @@ function addToExpression{C,V}(quad::GenericQuadExpr{C,V},c::Number,x::GenericQua
     quad
 end
 
-function addToExpression{N}(ex::Array{AffExpr,N}, c::Number, x::Array{AffExpr,N})
-    size(ex) == size(x) || error("Incompatible sizes: $(size(ex)) + $(size(x))")
+function chkdims(x,y)
+    (ndims(x) ≤ 2 &&
```

Comment:
I didn't want to touch tensor multiplication stuff, but I guess stuff like ``@addConstraint(m, X + Y .== Z)`` should be allowed.